File: 217-contains-duplicate/contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet<Integer> set = new HashSet<>();
        for(int n : nums){
            if(!set.contains(n)){
                set.add(n);
            }
            else{
                return true;
            }
        }
        return false;
    }
}

File: 217-contains-duplicate/contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        hashSet = set()
        for i in ((nums)):
            if i  in hashSet:
                return True
                
            hashSet.add(i)
        return False
# class Solution:
#     def containsDuplicate(self, nums: List[int]) -> bool:
#         hashSet = set()
#         for num in nums:
#             if num in hashSet:
#                 return True
#             hashSet.add(num)
#         return False